src/math/pow.c

```diff
@@ -0,0 +1,32 @@
+/* This is free and unencumbered software released into the public domain. */
+
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+/**
+ * @date   2017-10-01
+ * @author Samuel Sarle
+ * @see    http://libc11.org/math/pow.html
+ */
+
+double
+pow(double x, double y) {
+    double z = 1;
+    int i;
+    for (i=0;i<y;i++)
```

Comment:
Going forward, leave spaces after semicolon here. (I'll write up a style guideline so you won't need to guess.)

Reply:
Also, we use C11 (not C89), so you can (and should) write `for (int i = 0; i < y; i++)` instead of pre-declaring `i`